runtime/lua/vim/lsp/protocol.lua

```diff
@@ -898,6 +898,7 @@ end
 ---@param server_capabilities table Table of capabilities supported by the server
 ---@return table Normalized table of capabilities
 function protocol._resolve_capabilities_compat(server_capabilities)
+  vim.deprecate('protocal_resolve_capabilities', nil, '0.10')
```

Comment:
Shouldn't the version be `'1.0'`?

Reply:
no!

Reply:
(after `0.9.x` comes `0.10.0`, and then on until `0.100.0`; `1.0.0` has a very specific meaning (stability) which we are still some ways away from.)